### PR TITLE
bump version to 2.1.0 and update release notes

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by pforai** [comment]
Looks good!

File: RELEASE_NOTES [hunk 1]
```diff
@@ -3,16 +3,49 @@ For more detailed information, please see the git log.
...
 -----------------------
 
 feature + bugfix release
 - added example easyconfig files for 29 new software packages:
-    bsoft (#1353), Coot (#1400), Cuby (#1258), DSRC (#1242), Exonerate (#568), fastqz (#1242), FSA (#568), fqzcomp (#1242), GAMESS-US (#1153, #1406), Grep (#1308), Hadoop (#1418), Hoard (#1415), IMB (#1284), ISL (#1389), jemalloc (#1416), libdwarf (#1283), libelf (#1283), MPC (#1310), multitail (#1327), Pmw (#1403), Quip (#1242), rCUDA (#720), SCALCE (#1242), SMALT (#568), STREAM (#1332), worker (#1307), Xerces-C++ (#1370), XQilla (#1370), ZPAQ (#1242)
+    bsoft (#1353), Coot (#1400), Cuby (#1258), DSRC (#1242), Exonerate (#568), fastqz (#1242), FSA (#568),
+    fqzcomp (#1242), GAMESS-US (#1153, #1406), Grep (#1308), Hadoop (#1418), Hoard (#1415), IMB (#1284), ISL (#1389),
+    jemalloc (#1416), libdwarf (#1283), libelf (#1283), MPC (#1310), multitail (#1327), Pmw (#1403), Quip (#1242),
+    rCUDA (#720), SCALCE (#1242), SMALT (#568), STREAM (#1332), worker (#1307), Xerces-C++ (#1370), XQilla (#1370),
+    ZPAQ (#1242)
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: oops; we forgot to update this during eb/2.0.0, didn't we?

File: RELEASE_NOTES [hunk 2]
```diff
@@ -3,16 +3,49 @@ For more detailed information, please see the git log.
...
+    AFNI (#1322, #1521), BCFtools (#1492), getdp (#1518), gmsh (#1518), gtest (#1244), hanythingondemand (#1530),
+    mawk (#1369), Minimac (#815), Minimac3 (#1502), monty (#1548), Octave (#1563), pbs_python (#1530), pigz (#1036),
+    Pygments (#1536), pyhull (#1539), pymatgen (#1549), PyQt (#1322, #1521), Ray (#1494), requests (#1536),
+    seqtk (#1524), SIP (#1322, #1521), S-Lang (#1369), Spark (#1554), spglib (#1549), TINKER (#1465), tmux (#1369),
+    Xmipp (#1489)
+- added easyconfigs for new (Cray-specific) toolchains (#1538): CrayGNU, CrayIntel, CrayCCE
+    - initially supported software (using CrayGNU toolchains): CP2K, GROMACS, HPL, Python + numpy/scipy, WRF (#1558)
+- added new easyconfigs for existing toolchains: goolf/1.5.16, intel/2014.06
+- added additional easyconfigs for various supported software packages: version updates, different toolchains, ...
+    including GCC v5.1.0, OpenFOAM v2.3.1, R v3.1.3, PETSc/SLEPc v3.5.3, WIEN2k v14.1
```
Comment (fgeorgatos) [inline RELEASE_NOTES]: overall, this PR looks good; pity we did not add R/3.2.0 in this release (have it tested and working for several days)!

File: RELEASE_NOTES (hunk 1 above)
Comment (fgeorgatos) [inline RELEASE_NOTES]: ok, it was a long line... NM. similarly, consider L17 below.

Comment (boegel) [inline RELEASE_NOTES]: :+1:

**Comment by boegel** [comment]
Thanks for the review!

I'll merge this in, fork off a version branch, and bump the version in develop to `2.2.0dev`; release will happen later.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
going in, thanks for the review @fgeorgatos and @pforai!

**Pull request merged**